templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -68,7 +74,15 @@ Parameters:
   KeyPairName:
     Description: Public/private key pairs allow you to securely connect to your instance
       after it launches
-    Type: AWS::EC2::KeyPair::KeyName
+    Type: String
+    Default: ''
+  BastionHostRequired:
+    Default: "false"
+    AllowedValues:
+      - "true"
+      - "false"
+    Description: Whether to provision a Bastion host instance. (if true then an SSH Key Pair must be provided)
```

Comment:
Can we change this to:
> Whether to provision a Bastion host instance; if 'true', then you need to provide an SSH Key Pair
Also, the "Cluster nodes" group has a parameter with the following name/description:
> SSH keyname to use with the repository
> Named Key Pair name to use with this repository. The key should be imported into the SSM parameter store. (Optional)
To avoid confusion between this and the key pair required by Bastion, can we change that as well? Not sure what to change it to, though, as I'm not 100% sure what the **SSH keyname** is used for.
![Screen Shot 2020-05-06 at 2 27 36 pm](https://user-images.githubusercontent.com/5925362/81138617-ffab1780-8fa5-11ea-9ea3-c4e7daa7ecff.png)

Reply:
For "Cluster nodes" the description is fine as it is. This key is completely unrelated to the one we are interested in.